packages/devtools_app/lib/src/screens/app_size/app_size_screen.dart

```diff
@@ -619,18 +531,90 @@ class _DiffViewState extends State<DiffView>
       ),
     );
   }
+}
+
+class _AppSizeView extends StatelessWidget {
+  const _AppSizeView({
+    required this.title,
+    required this.treemapKey,
+    required this.treemapRoot,
+    required this.onRootChangedCallback,
+    required this.analysisTable,
+    required this.callGraphRoot,
+  });
+
+  final String title;
+
+  final Key treemapKey;
+
+  final TreemapNode treemapRoot;
+
+  final Function(TreemapNode?) onRootChangedCallback;
+
+  final Widget analysisTable;
+
+  final CallGraphNode? callGraphRoot;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(top: intermediateSpacing),
+      child: RoundedOutlinedBorder(
+        clip: true,
+        child: Column(
+          children: [
+            AreaPaneHeader(
+              title: Text(title),
+              maxLines: 2,
+              roundedTopBorder: false,
+              includeTopBorder: false,
+            ),
+            Expanded(
+              child: Split(
+                axis: Axis.vertical,
+                initialFractions: const [
+                  initialFractionForTreemap,
+                  initialFractionForTreeTable,
+                ],
+                children: [
+                  _buildTreemap(),
+                  OutlineDecoration.onlyTop(
+                    child: Row(
+                      children: [
+                        Flexible(
+                          child: analysisTable,
+                        ),
+                        if (callGraphRoot != null)
+                          Flexible(
+                            child: OutlineDecoration.onlyLeft(
+                              child: CallGraphWithDominators(
+                                callGraphRoot: callGraphRoot!,
+                              ),
+                            ),
+                          ),
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
 
   Widget _buildTreemap() {
```

Comment:
Should it be widget?

Reply:
declared the treemap directly in the build method instead to remove this helper